test/metrics/number_variables/4.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @ClassName: Hospital
+ * @Description: TODO
+ * @author: Bruce Young
+ * @date: 2020年02月02日 20:58
+ */
+public class Hospital {
+
+
+    private int x = 800;
+    private int y = 110;
+
+    private int width;
+    private int height = 606;
+
+    public int getWidth() {
+        return width;
+    }
+
+
+    public int getHeight() {
+        return height;
+    }
+
+    public int getX() {
+        return x;
+    }
+
+    public int getY() {
+        return y;
+    }
+
+    private static Hospital hospital = new Hospital();
+
+    public static Hospital getInstance() {
+        return hospital;
+    }
+
+    private Point point = new Point(800, 100);
+    private List<Bed> beds = new ArrayList<>();
+
+    private Hospital() {
+        if (Constants.BED_COUNT == 0) {
```

Comment:
Looks like you do not count `Constants.BED_COUNT` variable

Reply:
fixed